test/search-params.test.ts

```diff
@@ -227,4 +227,18 @@ describe('SearchParams', () => {
       'user_query=title%3Afoo+AND+collection%3Abar&aggregations=%5B%7B%22terms%22%3A%7B%22field%22%3A%22foo%22%2C%22size%22%3A10%7D%7D%2C%7B%22terms%22%3A%7B%22field%22%3A%22bar%22%2C%22size%22%3A7%7D%7D%5D';
     expect(queryAsString).to.equal(expected);
   });
+
+  it('properly generates a URLSearchParam with debugging enabled', async () => {
+    const query = 'title:foo';
+    const params = {
+      query,
+      debugging: true,
+    };
+    const urlSearchParam = SearchParamURLGenerator.generateURLSearchParams(
+      params
+    );
+    const queryAsString = urlSearchParam.toString();
+    const expected = 'user_query=title%3Afoo&debugging=true';
```

Comment:
Q: what do you think about `debugging` param as always true when present? or even on the client side browser level, we can use `&debug=1` or `&debug` so we can honor the limited real estate of an url.

Reply:
I think sticking with `debugging` makes sense because this is meant to be a passthrough to the PPS (which uses `debugging` and not `debug`). Probably easier for PPS testing / code clarity if the params match up?

I suppose the search service could be made to interpret `&debugging` as equivalent to `&debugging=true` though...

Reply:
as-is is fine. just noting these things, maybe we can tighten up our urls in the next couple of cycles..